lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+const isEmbroider = require('./is-embroider');
+
+// TODO: hard coding these seem unfortunate, especially vendor-static which is non-standard
+const FILES_TO_SKIP = /vendor|vendor-static/gi;
+
+module.exports = function updateManifestFromHtml(root) {
+  const pkg = JSON.parse(fs.readFileSync(root + '/package.json', 'UTF8'));
```

Comment:
should this be moved after the `isEmbroider` guard, as if that guard is truithy we will not use the `pkg`

Reply:
Not needed as `isEmbroider` is happening in `postBuild` hook.